integration/helpers/vite.ts

```diff
@@ -272,6 +293,55 @@ export const test = base.extend<Fixtures>({
     stop?.();
   },
   // eslint-disable-next-line no-empty-pattern
+  viteDevDeno: async ({}, use) => {
+    let stop: (() => unknown) | undefined;
+    await use(async (files, template) => {
+      let port = await getPort();
+      let cwd = await createProject(await files({ port }), template);
+
+      // Install dependencies as late as possible so Node won't accidentally import them and cause access denied errors in Deno on Windows
```

Comment:
this is my solution for this error, https://github.com/remix-run/remix/actions/runs/10672525296/job/29580157581